web/src/compose_ui.js

```diff
@@ -249,15 +250,15 @@ export function compute_placeholder_text(opts) {
     // For direct messages
     if (opts.private_message_recipient) {
         const recipient_list = opts.private_message_recipient.split(",");
-        const recipient_names = recipient_list
-            .map((recipient) => {
+        const recipient_names = new Intl.ListFormat(page_params.request_language).format(
```

Comment:
Why is this using `page_params.request_language`? I'm pretty sure we want to use the user's actual setting, `user_settings.default_language`, instead.